app/models/manageiq/providers/redhat/inventory_collection_default/infra_manager.rb

```diff
@@ -81,11 +81,13 @@ def disks(extra_attributes = {})
     def networks(extra_attributes = {})
       attributes = {
         :model_class                 => ::Network,
+        :manager_ref                 => [:hardware, :ipaddress, :ipv6address],
```

Comment:
@pkliczewski is it ok to override a key here?

Reply:
This is a key of network entity to select it uniquely. How that would work if `:ipv6address` is not there?

Reply:
it should be ok for full refresh (we will need to evaluate the targeted refresh queries, when targeted refresh is used here)

in general we allow nil in some keys, but the combination must be unique. (e.g. just hardware is not allowed to be nil, since that is the foreign key, graph refresh will throw error if there will be nil)

Reply:
btw. we should probably move this definition to baseclass then

Reply:
+1 for moving it [here](https://github.com/ManageIQ/manageiq/blob/master/app/models/manager_refresh/inventory_collection_default/infra_manager.rb#L6)

Reply:
Done, moved to https://github.com/ManageIQ/manageiq/pull/16619
Ran oVirt's and vmware's specs manually, they pass.